lib/flash_flow/deploy.rb

```diff
@@ -88,7 +88,10 @@ def merge_pull_requests
         if remote.nil?
           raise RuntimeError.new("No remote found for #{pull_request.head.repo.ssh_url}. Please run 'git remote add *your_remote_name* #{pull_request.head.repo.ssh_url}' and try again.")
         end
-        merge_or_rollback(remote, pull_request.head.ref, pull_request.number)
+
+        unless @github.has_label?(pull_request.number, @github.do_not_merge_label)
```

Comment:
@github already knew another label (`unmergeable`) and the `has_do_not_merge_label?` generalized with `has_unmergeable_label?` into `has_label?`